llvm/test/MC/ELF/cfi-sframe-cfi-escape-errors.s

```diff
@@ -0,0 +1,37 @@
+# RUN: llvm-mc --filetype=obj --gsframe -triple x86_64 %s -o %t.o 2>&1 | FileCheck %s
```

Comment:
If there are no errors (llvm-mc exits with 0), consider `-diagnostics` instead of `errors`.